lib/ui/view/dm_user/dm_user_viewmodel.dart

```diff
@@ -74,4 +74,93 @@ class DmUserViewModel extends BaseViewModel {
     );
     notifyListeners();
   }
+
+  Future<dynamic> showButtomSheet(BuildContext context) {
```

Comment:
Make use of `stacked_bottomSheet` service, and register a custom bottomSheet, if you don't know about it, read about it [here](https://pub.dev/packages/stacked_services#bottomsheet-service).

and ensure no material or Cupertino widget is imported in the viewmodel

Reply:
Done